packages/angular/projects/vcd/plugin-builders/src/lib/common/postcss-precalculate-rem.ts

```diff
@@ -0,0 +1,198 @@
+import * as postcss from 'postcss';
+import { PrecalculateRemOptions } from "./interfaces";
+
+const defaults: PrecalculateRemOptions & {unitPrecision: number, selectorBlackList: string[], mediaQuery: boolean} = {
+    rootValue: 16,
+    unitPrecision: 5,
+    selectorBlackList: [],
+    propList: ['font', 'font-size', 'line-height', 'letter-spacing'],
+    replace: true,
+    mediaQuery: false,
+    minRemValue: 0
+};
+
+export const postCssPlugin = postcss.plugin('postcss-rem-to-pixel', function(options: any) {
+    const opts = {
+        ...defaults,
+        ...options,
+    };
+    const remReplace = createRemReplace(opts.rootValue, opts.unitPrecision, opts.minRemValue);
+
+    const satisfyPropList = createPropListMatcher(opts.propList);
+
+    return function (css) {
+
+        css.walkDecls(function (decl, i) {
+            // This should be the fastest test and will remove most declarations
+            if (decl.value.indexOf('rem') === -1) return;
+
+            if (!satisfyPropList(decl.prop)) return;
+
+            // TODO: `decl.parent.selector` is present in postcss version v5.2.18
+            if (blacklistedSelector(opts.selectorBlackList, (decl.parent as { selector: string }).selector)) return;
+
+            // calc(0.5 * 1rem)
+            const value = decl.value.replace(remRegex, remReplace);
+
+            // if px unit already exists, do not add or replace
+            if (declarationExists(decl.parent, decl.prop, value)) return;
+
+            if (opts.replace) {
+                decl.value = value;
+            } else {
+                decl.parent.insertAfter(i, decl.clone({ value: value }));
+            }
+        });
+
+        if (opts.mediaQuery) {
+            css.walkAtRules('media', function (rule) {
+                if (rule.params.indexOf('rem') === -1) return;
```

Comment:
Better keep the common syntax with '{'  :
if() {
   return ...
}

Reply:
fixed in commit -> [lint: fix npm run lint](https://github.com/vmware/vcd-ext-sdk/pull/234/commits/e3b1fd23aaa9519c4cdb9aa88686213f01435ab7)